apis_core/apis_metainfo/templates/base.html

```diff
@@ -96,7 +96,7 @@
           </ul>
 
           <!-- Start user login submenu -->
-          <ul class="navbar-nav justify-content-end">
+          <ul class="navbar-nav justify-content-end mr-0">
             {% if user.is_authenticated %}
               <li class="nav-item dropdown ml-auto">
                 <a href="" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
```

Comment:
I tried integrating that, but a) I don't know exactly what it fixes (I didn't see any change on mobile) and b) I didn't find any documentation on `ml-m-auto`

Reply:
`m` is one of the breakpoints.

On small screens, the menu items are displayed vertically, neatly lined up – except for the login menu, which may get overlooked esp. when logged out and generally looks off because it's to the right, preceded by a long stretch of whitespace.

![Screenshot 2023-05-31 at 20-32-37 TITLE](https://github.com/acdh-oeaw/apis-core-rdf/assets/113020169/57ec70b5-5f0e-420f-b3a9-fd47877386be)

Removing the `ml-auto` class altogether fixes the problem as well.